src/domain/synthetics/orders/simulateExecuteOrderTxn.tsx

```diff
@@ -58,7 +69,7 @@ export async function simulateExecuteOrderTxn(chainId: number, p: SimulateExecut
         maxTimestamp: priceTimestamp,
       },
       ...(p.extraArgs ?? []),
-    ]),
+    ] as any),
```

Comment:
Not quite sure about this `any`s, as I'm a bit out of context with typechain. Is it possible to fix, or at least mitigate possible following issues if any?

Reply:
Good point, thanks. Made types more strict